src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -16,16 +16,18 @@ With log obfuscation rules, you can prevent certain types of information from be
 
 ## Requirements [#requirements]
 
-Log obfuscation is available as part of our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices). 
+The log obfuscation UI is available as part of our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices). 
 
 ## What is log obfuscation? [#overview]
 
-Our [log management service](/docs/logs/get-started/get-started-log-management) automatically masks number patterns that we identify as likely being sensitive items, such as credit card numbrers or Social Security numbers. Additionally, with our log obfuscation feature, you can set up additional data privacy for other personally-identifiable information by defining your own custom obfuscation rules right from the log management UI.
+Our [log management service](/docs/logs/get-started/get-started-log-management) automatically masks number patterns that we identify as likely being sensitive items, such as credit card or Social Security numbers. If you need additional obfuscation, one option is to adjust the configuration of the log forwarder you use (for example, our infrastructure agent). 
 
-With log obfuscation, you can define regular expressions matching your sensitive information, and then create rules to obfuscate that data. You can choose either to have sensitive information masked or hashed.
+But an easier option is to use our log obfuscation UI, available with [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices). This UI lets you set up log obfuscation rules directly from the log management UI, without lengthy manual configuration. You'll define regular expressions matching your sensitive information, and then create rules to obfuscate that data. You can choose either to have sensitive information masked or hashed.
```

Comment:
Recommend adding a sentence along the lines of "You can use our obfuscation API in NerdGraph, our GraphQL Explorer."